packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -336,7 +334,13 @@ Nimble already has a mechanism for clients to provide custom columns by deriving
 
 ### Additional unit APIs
 
-We considered adding attributes `prefix-field-name` and `suffix-field-name` which would allow those values to be populated with a dynamic field value rather than a constant string. This could help use cases where values have disparate types or units. We decided this is not necessary initially but could revisit in the future. See [this thread](https://github.com/ni/nimble/pull/1268#discussion_r1212385898) for more discussion.
+We considered adding attributes for `prefix` and `suffix` that could be used to specify any string to append to the front or back of the formatted number. We decided against this idea because it poses a localization challenge where units that are a suffix in one locale might be a prefix in another. Therefore, we decided to require units to be those supported by ECMAScript so that localization would be built in to the formatter.
```

Comment:
Do we have an example of units are a suffix in one locale and a prefix in another? If so, it might be good to just cite it here.